astropy/tests/__init__.py

```diff
@@ -4,3 +4,25 @@
 for writing tests, and general tests that are not associated with a
 particular package.
 """
+
+# NOTE: This is retained only for backwards compatibility. Affiliated packages
+# should no longer import `disable_internet` from `astropy.tests`. It is now
```

Comment:
It doesn't matter here as it's not processed by sphinx, but most of these backticks should be "``"